tests/unit/instance-initializers/violations-helper-test.js

```diff
@@ -1,22 +1,35 @@
 /* global sinon */
+
+import Application from '@ember/application';
 import { initialize } from 'dummy/instance-initializers/violations-helper';
 import { module, test } from 'qunit';
+import { run } from '@ember/runloop';
 
-let application;
 let sandbox;
 
-module('Unit | Instance Initializer | violations-helper', {
-  beforeEach() {
+module('Unit | Instance Initializer | violations-helper', function(hooks) {
+  hooks.beforeEach(function() {
+    this.TestApplication = Application.extend();
+    this.TestApplication.instanceInitializer({
+      name: 'initializer under test',
+      initialize
+    });
+    this.application = this.TestApplication.create({ autoboot: false });
+    this.instance = this.application.buildInstance();
+
     sandbox = sinon.sandbox.create();
-  },
+  });
+  hooks.afterEach(function() {
+    run(this.application, 'destroy');
+    run(this.instance, 'destroy');
 
-  afterEach() {
     sandbox.restore();
-  }
-});
+  });
 
-test('initializer sets a violationsHelper in the global scope', function(assert) {
-  initialize(application);
+  // Replace this with your real tests.
```

Comment:
This comment probably should be removed (the test below does have a valid assertion)